song/templates/deployment.yaml

```diff
@@ -46,8 +48,14 @@ spec:
                   key: id-authtoken
             - name: ID_REALIDS
               value: {{ .Values.appConfig.id.realIDs }}
-            - name: DCCSTORAGE_URL
-              value: {{ .Values.appConfig.dccStorage.url }}
+            - name: SCORE_URL
```

Comment:
just an fyi, you can use the extraEnv value in the values file, you don't have to modify the chart every time you need to add or remove an env var

Reply:
ok, DCCSTORAGE is obselete, so i figured id remove something that doesnt exist